example-frontend/pages/autoOffsetExactOutETH.tsx

```diff
@@ -0,0 +1,45 @@
+import { useSigner } from "wagmi";
+import { parseEther } from "ethers/lib/utils";
+import { ethers } from "ethers";
+import OffsetHelper from "../../deployments/polygon/OffsetHelper.json";
+
+export default function AutoOffsetExactOutETH() {
```

Comment:
Both `OutXXX` functions are reverting when being called. Maybe you know why?

Reply:
are you testing in Mumbai? tbh. The reason for that could be liquidity issues.. 
I've tested them on Polygon and there they both work..

Reply:
@GigaHierz yes, I tested in Mumbai.